lib/travis/logs/drain_consumer.rb

```diff
@@ -154,7 +137,11 @@ def dead?
           end
         else
           Travis.logger.info('acking empty or undecodable payload')
-          safe_ack(delivery_info.delivery_tag)
+          begin
+            ack(delivery_info.delivery_tag)
+          rescue Bunny::Exception => e
+            log_exception(e)
+          end
```

Comment:
This `begin`, `rescue` `log_exception` bit is what was intended with `safe_ack`, but if it's only being used in one place now, then whatever.